### DISCUSSION TITLE
How to paste into prompt?

### DISCUSSION THREAD

**Discussion by ariel-frischer** [jackMort/ChatGPT.nvim#25] [Q&A] [answered]
Usually paste with Command + V on macOS but this doesn't seem to work. I also can't go into normal mode while inside the prompt and use `p` to paste from register. Is this working for others? How? Super cool plugin btw!

**Comment by ben-wall** [comment]
It looks like he's working on a way to invoke the window with a vim range selected, so you might not need to.

**Reply by ariel-frischer** [reply]
Yea that would be useful too but I'm sure people would want to copy paste into the prompt as well.

**Reply by ben-wall** [reply]
agree

**Comment by getlost01** [comment, marked as answer]
To paste text in a terminal application or shell, you can try using the middle mouse button or the `Ctrl+Shift+V` keyboard shortcut. If these options don't work, you might need to use a different method to paste the text.

**Reply by ariel-frischer** [reply]
Oh, ok wierd that `command+v` is just working for me now, I guess this is a non-issue.
